bin/cmd.js

```diff
@@ -455,7 +455,8 @@ function runDownload (torrentId) {
 
     function openPlayer (cmd) {
       unref(cp.exec(cmd, function (err) {
-        if (err) return fatalError(err)
+        // Code 4 is returned by MPV when quiting
+        if (err && err.code !== 4) return fatalError(err)
```

Comment:
We shouldn't hard code an error code that is specific to MPV since this code path is used for all types of video players.

Reply:
Turns out that mpv always incorrectly exits with a non-zero code when killed by the parent process. We'll have to hardcode this :(